data_psi.go

```diff
@@ -241,7 +243,7 @@ func parsePSISectionHeader(i *astikit.BytesIterator) (h *PSISectionHeader, offse
 	h.PrivateBit = bs[0]&0x40 > 0
 
 	// Section length
-	h.SectionLength = uint16(bs[0]&0xf)<<8 | uint16(bs[1])
+	h.SectionLength = uint16(bs[0]&3)<<8 | uint16(bs[1])
```

Comment:
It seems that `&3` is not correct here, it should be `0xf`, since the specification says that section length is 12 bits long.

<img width="560" alt="Screenshot 2021-04-07 at 16 01 37" src="https://user-images.githubusercontent.com/2210994/113870466-8b32b700-97ba-11eb-8092-57822dd2c095.png">

Reply:
FYI: I always use this copy of spec: http://www.itu.int/rec/T-REC-H.222.0-201410-S/en

Reply:
On page 50 it says the section_length is only 10 bits:

>section_length – This is a 12-bit field, the first two bits of which shall be '00'. The remaining 10 bits specify the number of bytes of the section, starting immediately following the section_length field, and including the CRC. The value in this field shall not exceed 1021 (0x3FD).

Reply:
Fair enough, I missed that one. Can you please fix this one in muxer as well?